### PR TITLE
feat:implemented factorial chain function

### PR DESCRIPTION
**Pull request by hrithik-b01**
Hello everyone. I implemented a factorial chain array function. Please review the code

### CODE REVIEW

File: array.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * This function checks if an array is a factor chain or not
+ * Example of factor array - [2,4,8,16]
+ * Example of not a factor array - [2,4,8,28]
+ * @param {array} arr 
+ * @returns the result- True or false.
+ */
+
+function factrChain(arr){
```
Comment (KrithikaSuvarna01) [inline array.js:12]: Can you please give a context of what the code does

**Review by KrithikaSuvarna01 — CHANGES_REQUESTED** [review]
Can you please give a context of what the code does

File: array.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ * This function checks if an array is a factor chain or not
+ * Example of factor array - [2,4,8,16]
```
Comment (pr-th-m) [inline array.js:4]: It would be wonderful to add a proper formatting to array elements, do the same to the line below

**Review by pr-th-m — CHANGES_REQUESTED** [review]
Have a good day

File: array.js (hunk 2 above)
Comment (hrithik-b01) [reply array.js:4]: Sure, formatted the code and added additional comments

**Review by pr-th-m — APPROVED** [review]
Looks good

**Review by KrithikaSuvarna01 — APPROVED** [review]
LGTM..

**Review by NIREE23 — APPROVED** [review]

**Review by NIREE23 — APPROVED** [review]

**Pull request merged**